src/components/myGallery/MyGallery.jsx

```diff
@@ -0,0 +1,29 @@
+import './MyGallery.css';
+import ArtCard from '../artCard/ArtCard';
+import { useContext } from 'react';
+import { MemeContext } from '../../contexts/MemeContext';
+
+const MyGallery = (arrayOfFavs) => {
+    const {memes} = useContext(memeContext)
```

Comment:
It doesn't look like `memes` is being used at all in this file. You may not need to bring in Context here?

Reply:
We hadn't built out MyGallery yet, we did end up using it eventually :)